site/write_ups/2023/29-03-23_writeup.md

```diff
@@ -0,0 +1,127 @@
+# Data Ethics Club: [The Tech We Won't Build](https://2022.internethealthreport.org/episodes/the-tech-we-wont-build/)
+<!--Please don't edit the info panel below-->
+
+<!--
+
+```{admonition} What's this? 
+This is summary of Wednesday 28th July's Data Ethics Club discussion, where we spoke and wrote about the New Republic article [The Rise of Private Spies](https://newrepublic.com/article/161913/we-are-bellingcat-spooked-private-investigators) by Charlie Savage.
```

Comment:
Admonition is just a template so needs to be changed for each writeup (date and relevant link)!
```suggestion
This is summary of Wednesday 29th March's Data Ethics Club discussion, where we spoke and wrote about the podcast episode [The Tech We Won't Build](https://2022.internethealthreport.org/episodes/the-tech-we-wont-build/) where "[Laura Nolan](https://2022.internethealthreport.org/story/say-no-to-killer-robots-laura-nolan/) shares the story behind her decision to leave Google in 2018 over their involvement in Project Maven, a Pentagon project which used AI by Google."
```

Reply:
This is just an example of what you could write. I usually say something like "this is an article/blog ~link~ from this journal/news site written by these people who are academics/journalist etc.